en/docs/references/pricing.md

```diff
@@ -0,0 +1,229 @@
+
+          floor(1200/500) = 2 units = 2 steps
+
+Therefore the total number of steps for the reservation process is: 
+
+           4 steps + 2 steps = 6 steps
+
+We can calculate the **total number of steps for searching and reserving a flight** as follows:
+
+- **Search process**: Single execution of a search would cost a total of **2 steps**. 
+- **Reservation process**: Single execution of reservation process would cost a total of **6 steps**. 
+- **Total**: A single execution of the search and reservation process of a flight would cost an organization a total of **8 steps in total**.
+
+!!! note
+    The above cost calculation is dependent on the above architecture and design of the APIs.
+<br/>
+
+### Example 2: REST API Proxy
+An organization exposes a flight tracking API as a managed API on Choreo. This system involves several APIs and databases. The scenario and the APIs in the system are described below:
+
+#### Scenario
+
+To search for real-time flight data, a client will enter search criteria, and search through the web application. Upon receiving the request, the client application (web application) will make a call to the [Flights Status API](#flightStatusAPI), a managed API on Choreo, to search for flight data. Choreo will then invoke the backend API, the [Real-Time Flight API](#realtimeFlightAPI). The backend API will invoke a database to get the information and pass it to the client.
+
+#### Glossary
+
+- <a class="pricing" id="flighStatusAPI">Flight Status API</a>: An external-facing, managed API on Choreo. This API exposes the Real-Time Flight API on Choreo as a managed API.
+
+- <a class="pricing" id="realtimeFlightAPI">Real-TIme Flight API</a>: An unmanaged, external-facing API hosted outside of Choreo.
+
+- <a class="pricing" id="flightsDatabase">Flights Database</a>: A database that stores flight information.
+
+
+#### Get flight status
+
+![Get flight status](../assets/img/references/api-proxy-example1.png)
+
+Let’s see how the steps are calculated for the get flight status process:
+
+- As illustrated above, the client initiates a search request for flight data through the web application. 
+- The web application then makes **1 API call to the [Flight Status API](#flightStatusAPI)** in Choreo. This is counted as **1 step**.
+- The Flight Status API in Choreo then makes **1 call to the [Real-Time Flight API](#realtimeFlightAPI)** with the search criteria entered by the client. This is counted as **1 step**. 
+- The Real-Time Flight API then invokes the database to retrieve the information.
+
+- **Assuming the entire execution time for the get flight status process takes less than 500ms** (starting from making a call to Choreo, and Choreo making a call to the backend API),the get flight status process thereby results in a **total of 2 steps**.
```

Comment:
"starting from making a call to Choreo, and Choreo making a call to the backend API"

Since we mention what makes the call to the  backend API, it is better to mention what makes the call to Choreo as well.

Reply:
done